components/Banner/src/MDCBannerView.h

```diff
@@ -16,6 +16,14 @@
 
 #import "MaterialButtons.h"
 
+typedef NS_ENUM(NSInteger, MDCBannerViewLayoutMode) {
+  MDCBannerViewLayoutModeAutomatic,   // Layout is set automatically based on how elements are
+                                      // configured on banner view
+  MDCBannerViewLayoutModeSingleLine,  // All elements on the same line, only supports one button
```

Comment:
“Row” may be more clear throughout these apis and docs due to the conflict of “line” with text label’s “number of lines”. I might expect that a single line layout will also only allow a single line of text, while a single row layout may have multiple lines of text.

Reply:
Good suggestion. Will revise the name here.

Reply:
Added a TODO in b5d725fdd272ec093f273a732c8948206c06f345. Will open another PR to address it since a global replace will be needed and it is a naming refactor.

Reply:
Please make that change in this PR because these are new APIs (or if you need to refactor it, please do the refactoring in a separate PR).

Reply:
Addressed in 4ea43103067da9c8aee739e47de6a29ae21b36de.